### ISSUE TITLE
Prefix / Suffix feature request

### ISSUE THREAD

**Issue by borisrorsvort** [madeintandem/jsonb_accessor#173]
Would it be possible to introduce prefix/suffix options for attribute names in jsonb_accessor? This would allow developers to write more concise and readable code, particularly when dealing with multiple attributes.

For example, instead of writing:

```ruby
jsonb_accessor :voting_options,
               voting_options_type:        [:string, { default: VOTING_OPTIONS_TYPES[0], store_key: :type }],
               voting_options_min_votes:   [:integer, { store_key: :min_votes }],
               ...

```

we could have: 

```ruby
jsonb_accessor :voting_options,
               type:        [:string, { default: VOTING_OPTIONS_TYPES[0], suffix: true }],
               ...
```

This addition would not only streamline code but also maintain readability and ease of maintenance.

I understand the complexities involved in maintaining an open-source project and adding new features. If this suggestion aligns with your vision for the gem, I'd be delighted to see it implemented.

**Comment by haffla** [comment]
Hey @borisrorsvort,

thanks. Sure I would welcome such addition to the code if it only changes the name of the getter and setter methods.

Your above example would generate

- `voting_options_type`
- `voting_options_type=`

instead of

- `type`
- `type=`

correct?

**Comment by borisrorsvort** [comment]
yes! but it would still be stored as type inside the json on the column voting_options.
